src/map/pc.c

```diff
@@ -4408,27 +4409,31 @@ int pc_paycash(struct map_session_data *sd, int price, int points)
 	if( points > price )
 	{
 		ShowWarning("pc_paycash: More kafra points provided than needed (price=%d, points=%d, account_id=%d, char_id=%d).\n", price, points, sd->status.account_id, sd->status.char_id);
```

Comment:
Why should it print a warning here? If the free points are more than the price, simply use the required amount of points and leave the excess unaffected?